dissect/fat/fat.py

```diff
@@ -294,6 +294,14 @@ def __repr__(self):
     def path(self):
         return "\\".join([self.parent.path if self.parent else "", self.name]).lstrip("\\")
 
+    @cached_property
+    def nblocks(self) -> int:
+        return len(list(self.fs.fat.chain(self.cluster)))
```

Comment:
Why not just divide `entry.size / fs.cluster_size` in `dissect.target`? Then this entire PR is not needed.

Reply:
would still be nice to have it in the interface